### PR TITLE
Introduce rhel9 profile stubs

### PR DESCRIPTION
**Pull request by matejak**
Many rules are already applicable, so this PR introduces RHEL8 profiles for RHEL9 in a form that will allow the content to compile.

Profiles are marked as drafts, but they have not been examined closely - it may be that some rules will fall out, and will be substituted by brand new rules.

### CODE REVIEW

**Comment by matejak** [comment]
Do we have some sort of automated CCE assignment functionality in the project?

**Comment by cipherboy** [comment]
@matejak said:
> Do we have some sort of automated CCE assignment functionality in the project?

If you want one and it doesn't yet exist, you should be able to create one fairly easy by changing the `autorefer.py` (that PR I opened and subsequently closed) to grab CCIs out of the file, remove them, and add them into rules based on entries in a profile. Just a thought. :-) 

Maybe the tooling could be generalized similar to `fix_rules.py` and include? I dunno.

File: linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_warn_age_login_defs/rule.yml
Comment (cipherboy) [inline linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_warn_age_login_defs/rule.yml:23]: Note that this assignment conflicts with #7069 just FYI.

Comment (matejak) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_warn_age_login_defs/rule.yml:23]: Luckily we have tests for duplicate CCEs. When we get more CCEs, then I plan to rebase to master branch except that I will put new CCEs to the appropriate file, and let the test suite to tell me where are the conflicts.

Comment (cipherboy) [reply linux_os/guide/system/accounts/accounts-restrictions/password_expiration/accounts_password_warn_age_login_defs/rule.yml:23]: Sounds good :) I don't know which will merge first, so whichever does, just gotta make sure we rebase after one lands so the tests catch it.

File: products/rhel9/profiles/cis.profile
Comment (yuumasato) [inline products/rhel9/profiles/cis.profile:14]: Small typo in RHEL version, which I think does not block this PR.

Comment (matejak) [reply products/rhel9/profiles/cis.profile:14]: It is not a typo - it is indeed based on the RHEL8 CIS for the time being.

**Review by yuumasato — APPROVED** [review]
LGTM

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Comment by evgenyz** [comment]
@openscap-ci ok to test

**Comment by evgenyz** [comment]
@openscap-ci test this, please

**Comment by evgenyz** [comment]
@openscap-ci test this, please

**Comment by yuumasato** [comment]
@openscap-ci test this please

**Pull request merged**
